### PR TITLE
Get blame for the target branch of the pull request

### PR DESCRIPTION
**Pull request by agubler**
Get blames for the target branch of the PR instead of assuming `master`

### CODE REVIEW

**Comment by agubler** [comment]
Also removed a number of trailing commas

File: server.js [hunk 1]
```diff
@@ -93,6 +93,7 @@ app.post('/', function(req, res) {
         data.repository.html_url, // 'https://github.com/fbsamples/bot-testing'
         data.pull_request.number, // 23
         data.pull_request.user.login, // 'mention-bot'
+        data.pull_request.base.ref,
```
Comment (vjeux) [inline server.js]: can you put `// 'master'` next to it, it's not clear what base.ref is

File: __tests__/github-mention-test.js [hunk 2]
```diff
@@ -54,16 +56,16 @@ describe('Github Mention', function() {
     expect(parsed).toEqual([
       {
         path: 'Libraries/Components/MapView/MapView.js',
-        deletedLines: [ 74 ],
+        deletedLines: [ 74 ]
```
Comment (vjeux) [inline __tests__/github-mention-test.js]: can you put back the trailing commas, we require them inside of fb :P

File: __tests__/github-mention-test.js [hunk 3]
```diff
@@ -28,7 +28,9 @@ describe('Github Mention', function() {
         mentionBot.guessOwnersForPullRequest(
           'https://github.com/facebook/react-native',
           i,
-          'mention-bot'
+          'mention-bot',
+          'master',
+          {}
```
Comment (vjeux) [inline __tests__/github-mention-test.js]: can you put `// config`

File: mention-bot.js [hunk 4]
```diff
@@ -343,7 +344,7 @@ function guessOwnersForPullRequest(
   var blames = {};
   files.forEach(function(file) {
     var path = file.path;
-    var blame = fetch(repoURL + '/blame/master/' + path);
+    var blame = fetch(repoURL + '/blame/' + targetBranch + '/' + path);
```
Comment (vjeux) [inline mention-bot.js]: nice!

**Comment by vjeux** [comment]
If you fix the nits i'll pull it in, thanks!

File: __tests__/github-mention-test.js (hunk 2 above)
Comment (agubler) [inline __tests__/github-mention-test.js]: no problem, do you mind if I ask why?

**Comment by vjeux** [comment]
if you add another argument, the blame looks something like

+ previousLine,
+ newLine

So you are now associated the blame for the previous line even though you did not change it. Also it clobbers the pull request because it highlights lines that did not change.

Finally, small thing but it's easier to move things around as you don't need to remove the comma from the last line and add it back to the other place.

**Comment by vjeux** [comment]
We are also pushing trailing comma for function calls and arguments to be part of the javascript standard. It's currently in stage 2.

**Comment by kaelig** [comment]
Thank you!

**Pull request merged**